delivery_carrier_warehouse/views/stock_warehouse.xml

```diff
@@ -8,7 +8,7 @@
             <field name="partner_id" position="after">
                 <field
                     name="delivery_carrier_id"
-                    domain="['|', ('company_id', '=', False), ('company_id', '=', company_id.id)]"
+                    domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]"
```

Comment:
```suggestion
                    domain="['|', ('company_id', '=', False), ('company_id', 'in', company_ids)]"
```
no?

Reply:
I am afraid not. I can only find `company_id` at https://github.com/odoo/odoo/blob/14.0/addons/stock/models/stock_warehouse.py#L34 and could not find `company_ids` in any inherited `stock.warehouse` of odoo/addons.